mod/metric_answer_lookup/set/type_plus_right/metric_value/value.rb

```diff
@@ -1,3 +1,5 @@
+card_accessor :unknown
```

Comment:
the +Unknown card is a field of the +value and not the answer?  Why is that?

Also, why implement it as a pointer rather than a boolean?